src/main/java/Interface/Menu.java

```diff
@@ -7,16 +7,19 @@ public class Menu {
     private static Scanner scanner;
 
     public static void showMainMenu() { // this method displays the main menu to the user
-        System.out.println("1. Show all books.");
-        System.out.println("2. Add a new book.");
+        System.out.println("1. Show all books");
+        System.out.println("2. Add a new book");
+        System.out.println("3. Register");
+        System.out.println("4. Login");
+        System.out.println("5. Exit");
     }
 
     public static int getUserInput() {
```

Comment:
Rename this method to something like getMenuChoice() for now

Reply:
done